### ISSUE TITLE
Patterns API: Documentation

### ISSUE THREAD

**Issue by ramonjd** [Automattic/wp-calypso#44635]
When we publish the call for testing, let's provide some documentation in the Field Guide.

We should cover: 

- Patterns Toolkit PHP lib
- Relevant API endpoint(s)
- design source site to translation workflow
- pattern_meta (e.g. pbxlJb-zn-p2)

**Comment by ramonjd** [comment]
This task was supposed to be part of Phase 2 (pbAok1-1uA-p2#comment-3182), but we're bringing into our team's project board

**Comment by ramonjd** [comment]
We have PCYsg-t0i-p2 🙇 

It would be great to have a section on how to get source site patterns translated using `makepot/glotpress-import` 

Or do they run in an async job now?

**Comment by andrewserong** [comment]
@ramonjd I'd drafted this based on @lsl's post in Ganon, but hadn't hit publish yet! (PCYsg-t0E-p2)
It still needs to have the parent page set correctly, I was having trouble using the parent page selector, so just need to update that.

As a separate task we'll need to create an async job to handle this, there was a card in Up Next, I've just moved it over to Up Next in Patterns. I'd love to work on setting up the async job when we can spare the time!

**Comment by ramonjd** [comment]
> It still needs to have the parent page set correctly, I was having trouble using the parent page selector, so just need to update that.

Oh, great! Thanks for the update.

> As a separate task we'll need to create an async job to handle this, there was a card in Up Next, I've just moved it over to Up Next in Patterns

All good. An async job sounds like it would save people a bunch of time. For now, so long as folks know how to get their patterns up and running and translated/re-translated is enough I reckon.

>  I'd love to work on setting up the async job when we can spare the time!

🙇 We can plan it in. It'd be a worthy feature

**Comment by andrewserong** [comment]
Tiny update: the parent page picker in Gutenberg doesn't seem to play nicely when there are a _lot_ of pages to choose from, so I managed to hook up the parent page properly using the classic editor. Side project idea: propose a page picker modal for Gutenberg to support large knowledge-bases.

We now have:

* Parent page (PCYsg-t0i-p2) 
* Translation overview (PCYsg-t0E-p2)
* API endpoint and metadata (PCYsg-uUe-p2)

Still to be done:

* [x] Dedicated page for how the API endpoint works, and describing how `pattern_meta` metadata works (perhaps copy over from (pbxlJb-zn-p2)
* [x] ~Dedicated page for block patterns source sites, from a more designer-friendly perspective, with screenshots~ (borrowing perhaps from pcjTuq-3s-p2 and pcjTuq-Z-p2) — instead of a dedicated page, I've updated the parent page (PCYsg-t0i-p2) and fleshed out the section on block patterns source sites a little, with links to the relevant P2 posts for info relevant to designers, along with a link to the Gutenboarding site designs page.
